### PR TITLE
Standardising the naming of data dictionaries

### PR DESCRIPTION
**Pull request by matentzn**
- renamed vukuzazi to vz and genomics-england to ge (metadata, template and owl)
- Changed the makefile COHORTS variable to instead index the metadata directory. This makes updating the makefile (from the IHCC dev side) unnecessary.
- Fixed an error in prefixes that still listed the old KOGES
- updated all data dictionaries accordingly

Note: I am a bit surprised though that maelstrom is not rebuilt anymore as part of the process. Why is that? @beckyjackson Shouldn't we delete it then from the list of data dictionaries altogether?

### CODE REVIEW

**Comment by matentzn** [comment]
- [ ] Remember to update monarch.owl
- [ ] Remember to offer equivalencies.owl as a separate release product

File: Makefile [hunk 1]
```diff
@@ -58,6 +58,9 @@ TEMPLATES := $(foreach C,$(COHORTS),templates/$(C).tsv)
 # OWL file in the build directory for all cohorts (contains xrefs)
 ONTS := $(foreach C,$(COHORTS),build/$(C).owl)
 
+xx:
+	echo $(COHORTS)
```
Comment (jamesaoverton) [inline Makefile:62]: I would use `make -pn | grep COHORTS`

**Review by jamesaoverton — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (matentzn) [reply Makefile:62]: the xx task should not be there at all. Mistake in haste (just needed during dev). Thanks for tip!

**Comment by matentzn** [comment]
@beckyjackson Let me know if you see any problems with this - you know the whole pipeline better than me :) Also the question re: maelstrom (how it should be updated).

**Comment by matentzn** [comment]
@beckyjackson I will merge this now, but if you can, please review this pull request anyways; I just want to be sure this did not break anything on your end. :) Thanks!

**Pull request merged**